clang/lib/Driver/Driver.cpp

```diff
@@ -4341,6 +4341,9 @@ class OffloadingActionBuilder final {
       StringRef LibSuffix = isMSVCEnv ? ".obj" : ".o";
       SmallVector<DeviceLibOptInfo, 5> sycl_device_wrapper_libs = {
           {"libsycl-crt", "libc"},
+          {"libsycl-itt-user-wrappers", "libc"},
+          {"libsycl-itt-compiler-wrappers", "libc"},
+          {"libsycl-itt-stubs", "libc"},
```

Comment:
What is the expectation for controlling these?  Should they be able to be removed from the device compilation with `-fno-sycl-device-lib=libc`, or should they even be controlled at all?

Reply:
To be honest, I've gone with `libc` simple because I thought it worse to leave `itt` libraries beyond any filtering group. A better solution could've been to add a dedicated control group for these, but that would've been a change to CLI with all ensuing approval & documentation requirements.

To get back to the point, I definitely dislike the `libc` approach. Would it be OK to link `itt` libraries unconditionally, leaving them invisible even to `-fno-sycl-device-lib=all`?

Reply:
As a reference, the `itt` libraries on the OpenMP side of the equation are unconditional.  I don't know if it is OK to make the same assumption here.

Reply:
Switched to "unaffected by `-fno-sycl-device-lib=all`" in https://github.com/intel/llvm/pull/4615/commits/cb0a1ea2ef0047e22e23e16b67f74ea804f64ba8 (and then some)